jbmc/regression/jbmc-strings/StringSubstringConstantEvaluation2/test3.desc

```diff
@@ -0,0 +1,9 @@
+CORE symex-driven-lazy-loading-expected-failure
```

Comment:
Any idea why the xfail?

Reply:
When using symex driven lazy loading it seems that jbmc does not fully honor the `--property` option as several additional VCCs are generated (e.g. null pointer checks). Those regression tests though check that a specific VCC is not simplified. This is currently done by selecting that VCC via `--property` and then checking that jbmc prints `Generated 1 VCC(s), 1` remaining after simplification. If there would be more VCCs we could not check that this one VCC is the one that is not simplified.